src/types/systemIntake.ts

```diff
@@ -79,6 +79,11 @@ export type SystemIntakeForm = {
   createdAt: DateTime | null;
   archivedAt: DateTime | null;
   lcid: string;
+  lcidExpiration: DateTime | null;
+  lcidScope: string;
```

Comment:
If the value can be null in the database, shouldn't this be string or null?

Reply:
`lcidScope` will always be a string in the frontend app. If the API returns a falsy value, we convert it to a string. Form elements give errors when toggling between uncontrolled and controlled elements.